src/effects/effectchainmanager.cpp

```diff
@@ -188,10 +188,12 @@ void EffectChainManager::loadEffectChains(
             QDomElement chainElement = chainNode.toElement();
             EffectChainPointer pChain = EffectChain::createFromXml(
                     m_pEffectsManager, chainElement);
-            EffectChainSlotPointer pChainSlot = pRack->getEffectChainSlot(i);
-            if (pChainSlot) {
-                pChainSlot->loadEffectChain(pChain);
-                pChainSlot->loadChainSlotFromXml(chainElement);
+            if (!pChain->name().isEmpty()) { // skip unnamed = ejected chains.
```

Comment:
I think the chain state should be loaded if it is found in the effects.xml file. I don't understand why the name being in the XML file is important for that.

We now have the issue that chains have both a name and an ID and it is not clear why. What is the purpose of having both? Do we need both? Why not just have a name?